gridfs/__init__.py

```diff
@@ -618,14 +628,15 @@ def upload_from_stream(self, filename, source, chunk_size_bytes=None,
            Added ``session`` parameter.
         """
         with self.open_upload_stream(
-                filename, chunk_size_bytes, metadata, session=session) as gin:
+                filename, chunk_size_bytes, metadata, session=session) as gin:  # type: GridIn
```

Comment:
The type: GridIn comment shouldn't be needed anymore.

Reply:
Confirmed and removed